.goreleaser.yaml

```diff
@@ -16,18 +16,19 @@ builds:
   - goos: linux
     goarch: arm64
   ldflags:
+  - -X github.com/bitrise-io/envman/version.Version={{ .Version }}
   - -X github.com/bitrise-io/envman/version.Commit={{ .FullCommit }}
   - -X github.com/bitrise-io/envman/version.BuildNumber={{ index .Env "BITRISE_BUILD_NUMBER" }}
 
 archives:
 # GitHub release should contain the raw binaries (no zip or tar.gz)
 - format: binary
   # Name format should match the curl install script
-  name_template: "{{ .ProjectName }}-{{ .Os }}-{{ .Arch }}"
-  replacements:
```

Comment:
Deprecated and removed, no longer works.

Reply:
Did change from uppercase to lowercase in case of Darwin/Linux?

Reply:
No, the `replacements` option got deprecated and the suggested new way of doing things is with the `name_template` + Go templating

Reply:
Right, missed the "title" part from the template.